arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -568,6 +524,11 @@ class ffmpeg_video_input::priv
       // Continue to make seek request further back until we land at a frame
       // that is before the requested frame.
       frame_ts -= this->f_backstep_size * this->stream_time_base_to_frame();
```

Comment:
@chetnieter it looks like `f_backstep_size` is set to `-1`, but this code is subtracting this multiple from the `frame_ts`, so that double negative means it is actually increasing `frame_ts`.  Is this a bug or am I misunderstanding how this works?

Reply:
It is reset in `open` [here](https://github.com/Kitware/kwiver/blob/master/arrows/ffmpeg/ffmpeg_video_input.cxx#L256) or [here](https://github.com/Kitware/kwiver/blob/master/arrows/ffmpeg/ffmpeg_video_input.cxx#L261). Perhaps I should do away with the else statement and just initialize it to 12.

Reply:
Thanks, I should have searched the whole file, I was just looking at the diff.  It makes sense.  You could just initialize to 12, unless there is some special where you need to check if it is non initialized.  Either way, it's not important for this PR.